<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiKinect TCPIP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_Viewer">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QVTKWidget" name="qvtkwidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_TCPIP">
   <property name="windowTitle">
    <string>updated from TCPIP</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_closeall">
       <property name="text">
        <string>Close ALL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_openall">
       <property name="text">
        <string>Open ALL</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_IP">
       <property name="text">
        <string>myIPlabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_ClientState">
       <property name="text">
        <string>Client : ?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_graball">
       <property name="text">
        <string>Grab ALL</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_updateGit">
       <property name="text">
        <string>UpdateFromGit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Kinect">
   <property name="windowTitle">
    <string>pippo</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="groupBox_Kinect1">
        <property name="title">
         <string>Kinect 1</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_Close_kin1">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="comboBox_KinectSerials_kin1"/>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_status_kin1">
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="pushButton_Grab_kin1">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Grab</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_Open_kin1">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox_pipeline_kin1">
           <item>
            <property name="text">
             <string>Cpu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OpenGL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OpenCL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cuda</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_Kinect2">
        <property name="title">
         <string>Kinect 2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="5" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="comboBox_KinectSerials_kin2"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_status_kin2">
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_Open_kin2">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="pushButton_Grab_kin2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Grab</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_Close_kin2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox_pipeline_kin2">
           <item>
            <property name="text">
             <string>Cpu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OpenGL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OpenCL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cuda</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Libfreenect</string>
       </property>
       <widget class="QComboBox" name="comboBox_log">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>85</width>
          <height>27</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Debug</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Info</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Warning</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Error</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_openall</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_Open_kin1</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_openall</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_Open_kin2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_closeall</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_Close_kin2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_closeall</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_Close_kin1</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_graball</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_Grab_kin1</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_graball</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_Grab_kin2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenAllDevices()</slot>
  <slot>CloseAllDevices()</slot>
  <slot>GrabAllDevices()</slot>
 </slots>
</ui>
